### DISCUSSION TITLE
Display HTTP request method in navigation sidebar

### DISCUSSION THREAD

**Discussion by ivanmudarth** [squidfunk/mkdocs-material#3124] [Q&A]
Hello,

I was wondering if it is possible to display an HTTP request method (GET, POST, PUT etc.) on the left-hand navigation sidebar like in the image below:

![image](https://user-images.githubusercontent.com/33183884/137786314-0138afbe-0154-4db0-a128-c28d151a8180.png)

If not, is there some work around to create this effect? Or could this be a possible feature in the future?

**Comment by squidfunk** [comment]
It's currently not possible. In order to support this, the [Table of Contents](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#table-of-contents) extension (if icons should be defined for headings) or the navigation handling of MkDocs (if they should be defined for pages) would need to handle icons somehow. There are some issues/discussions related to this topic.
